storage/mongodb/files/scripts/test-mongo-with-pbench.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
I would fully omit this file and go with test-mongo.sh which can be feed to pbench-user-benchmark --config="testmongo" -- ./test-mongo.sh $1 $2 $3

Reply:
Then where do you want me to put the logic for pbench-registration and copy-resuts? In the playbook?